link/src/main/java/com/stripe/android/link/ui/cardedit/CardEditViewModel.kt

```diff
@@ -161,19 +160,4 @@ internal class CardEditViewModel @Inject constructor(
                 } as T
         }
     }
-
-    sealed class Result : Parcelable {
```

Comment:
Moved into PaymentDetailsResult to be used by both `CardEditViewModel` and `PaymentMethodViewModel` when connecting a new bank account.